src/test02/app.html

```diff
@@ -67,18 +67,18 @@
 			        		
 			        		<div class="row bg-gray">
 			        			<div class="col-xs-6">
-			        				<label id="transfer-amount">Amount to transfer:</label>
+			        				<label for="amount" id="transfer-amount">Amount to transfer:</label>
 			        			</div>
 			        			<div class="col-xs-6">
-				        			<input class="form-control" type="text" value="{{300 | currency}}">
+				        			<input id="amount" class="form-control" type="text" value="{{300 | currency}}">
 			        			</div>
 			        		</div>
 			        		<div class="row bg-gray">
 			        			<div class="col-xs-6">
-			        				<label id="transfer-frequency">Frequency:</label>
+			        				<label for="frequency" id="transfer-frequency">Frequency:</label>
 			        			</div>
 			        			<div class="col-xs-6">
-			        				<select class="form-control">
+			        				<select id="frequency" class="form-control">
 									  <option>Every day</option>
 									  <option>Weekly</option>
 									  <option>Bi-weekly</option>
```

Comment:
Test 2 was a bit of a challenge as I have never dealt with accessibility.

Some sources I used were:
https://webaccess.berkeley.edu/resources/tips/web-accessibility
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role
http://www.webhostingsearch.com/articles/25-ways-to-make-your-site-more-accessible.php

After reading through these sources and app.html, I came to the conclusion that I should attach label elements to their relevant text, and use the ARIA role "alert" to indicate to the user that an important and time-sensitive message has been displayed.

I haven't tested this solution with a screen reader yet but I am certainly curious to find out whether this is the correct solution or not.